plugins/content/createmenulink/createmenulink.php

```diff
@@ -54,32 +54,33 @@ public function onContentPrepareData($context, $data)
 			return true;
 		}
 
-			list($component, $view) = explode('.', $context);
-
-			$menu      = JFactory::getApplication()->getMenu('site');
-			$menuItems = $menu->getItems(
-							'link',
-							'index.php?option=' . $component . '&view=' . $view . '&id=' . $data->id
-						);
-
-			if (!empty($menuItems))
-			{
-				$data->menuid    = $menuItems[0]->id;
-				$data->menutitle = $menuItems[0]->title;
-				$data->menualias = $menuItems[0]->alias;
-				$data->menutype  = $menuItems[0]->menutype;
-				$data->parent_id = $menuItems[0]->parent_id;
-				JHtml::_('jquery.framework', false);
-				JHtml::_('script', 'media/plg_content_createmenulink/parentitem.js');
-			}
-
-			if (empty($menuItems))
-			{
-				JHtml::_('jquery.framework', false);
-				JHtml::_('script', 'media/plg_content_createmenulink/copytitle.js');
-			}
+		$jinput    = JFactory::getApplication()->input;
+		$component = $jinput->get('option');
+		$view      = $jinput->get('view');
 
-		return true;
+		$menu      = JFactory::getApplication()->getMenu('site');
+		$menuItems = $menu->getItems(
+			'link',	'index.php?option=' . $component . '&view=' . $view . '&id=' . $data->id
+		);
+
+		if (!empty($menuItems))
+		{
+			$data->menuid    = $menuItems[0]->id;
+			$data->menutitle = $menuItems[0]->title;
+			$data->menualias = $menuItems[0]->alias;
+			$data->menutype  = $menuItems[0]->menutype;
+			$data->parent_id = $menuItems[0]->parent_id;
+			JHtml::_('jquery.framework', false);
+			JHtml::_('script', 'media/plg_content_createmenulink/parentitem.js');
+		}
+
+		if (empty($menuItems))
+		{
+			JHtml::_('jquery.framework', false);
```

Comment:
No needs to write `JHtml::_('jquery.framework', false);` two times. Also you are checking `empty($menuItems)` two times. You may just use `else` as `$menuItems` will be either empty or not empty.